laptop-specific/battery.md

```diff
@@ -30,6 +32,22 @@ Once you've finally gotten your DSDT patched and battery working in macOS, it's
 
 ## Dual Battery
 
-Acidanthera has a guide on how to handle laptops with dual-batteries: [Link](https://github.com/acidanthera/VirtualSMC/blob/master/Docs/Dual%20Battery%20Support.md)
+Because macOS does not properly support systems with dual-batteries, you have to merge the two batteries in ACPI.
+
+Refer to this Acidanthera documentation for information on how to handle dual-battery laptops: [Link](https://github.com/acidanthera/VirtualSMC/blob/master/Docs/Dual%20Battery%20Support.md)
+
+## Cycle Count
+
+Some manufacturers, such as HP, supply cycle count information within their EC registers, however do not support the ACPI 4.0 specification's `_BIX` method within their firmware to allow for this information to be queried. In the past, Rehabman's ACPIBatteryManager employed a hack to support cycle counts on firmwares which do not have a `_BIX` method, however with SMCBatteryManager this is no longer supported and a `_BIX` method must be manually implemented. 
```

Comment:
Two long sentences in a row, break it up a bit.
`Some manufacturers, such as HP, already supply cycle count. However, it is not exposed within the `_BIX` method, or the `_BIX` method is not implemented. ...`
I'd probably replace `do not support the ACPI 4.0 spec... with either the above or something else.